multi-class-classification/recipe.yaml

```diff
@@ -0,0 +1,69 @@
+# `recipe.yaml` is the main configuration file for an MLflow Recipe.
+# Required recipe parameters should be defined in this file with either concrete values or
+# variables such as {{ INGEST_DATA_LOCATION }}.
+
+# Variables must be dereferenced in a profile YAML file, located under `profiles/`.
+# See `profiles/local.yaml` for example usage. One may switch among profiles quickly by
+# providing a profile name such as `local` in the Recipe object constructor:
+# `p = Recipe(profile="local")`
+#
+# NOTE: YAML does not support tabs for indentation. Please use spaces and ensure that all YAML
+# files are properly formatted.
+
+recipe: "classification/v1"
+# Specifies the name of the column containing targets / labels for model training and evaluation
+target_col: "class"
+# Specifies the value of `target_col` which is considered the positive class: red wine in this case.
+#positive_class: 1
+# Sets the primary metric to use to evaluate model performance. This primary metric is used
+# to sort MLflow Runs corresponding to the recipe in the MLflow Tracking UI
+primary_metric: "f1_score_macro"
```

Comment:
`f1_score_macro` is less well known compared to just `f1_score`. Is it really necessary here to use it?
Don't we already turn on auto data balancing for imbalanced dataset?